Cargo.toml

```diff
@@ -11,12 +11,14 @@ repository = "https://github.com/Lokathor/tinyvec"
 
 [dependencies]
 tinyvec_macros = { version = "0.1", optional = true }
+# Provides `Serialize` and `Deserialize` implementations
+serde = { version = "1.0", optional = true, default-features = false }
 
 [features]
 default = []
 
 # Provide things that utilize the `alloc` crate, namely `TinyVec`.
-alloc = ["tinyvec_macros"]
+alloc = ["tinyvec_macros", "serde/alloc"]
```

Comment:
Unfortunately optional-features of optional-deps don't work, activating `alloc` will activate `serde + serde/alloc`. Though, is `serde/alloc` needed? It seems like the `TinyVec` impls don't use it.

Reply:
Hm, alright.